tests/pipeline/test_mida_trainer.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import pytest
+from numpy import testing
+from sklearn.datasets import make_blobs
+from sklearn.linear_model import LogisticRegression
+from sklearn.preprocessing import MinMaxScaler
+
+from kale.pipeline.mida_trainer import MIDATrainer
+
+N_SAMPLES = 200
+
+
+@pytest.fixture(scope="module")
+def toy_data():
```

Comment:
It would be better to introduce this toy data, including its dimensions. We can consider how to reuse it for other related methods.